src/WebJobs.Extensions/Extensions/Timers/Listener/TimerListener.cs

```diff
@@ -110,15 +110,18 @@ public async Task StartAsync(CancellationToken cancellationToken)
 
             if (ScheduleStatus == null)
             {
+                var defaultDateTime = new DateTime(1, 1, 1, 0, 0, 0, DateTimeKind.Local);
+
                 // no schedule status has been stored yet, so initialize
                 ScheduleStatus = new ScheduleStatus
                 {
-                    Last = default(DateTime),
-                    Next = _schedule.GetNextOccurrence(now)
+                    Last = defaultDateTime,
+                    Next = _schedule.GetNextOccurrence(now),
+                    LastUpdated = defaultDateTime
```

Comment:
Would it be safe to make this change? I wonder some host code is relying on this property to be null to execute certain code path and we will break it with this change.

Reply:
I tested this E2E with the host and the host does not expect them to be null, doesn't make changes to the value. Without this change the kind is **unspecified** for LastUpdated. I'm not sure where else LastUpdated is being set but so far looks like this is a fine change to make